src/System.Management.Automation/FormatAndOutput/common/FormatViewGenerator_Complex.cs

```diff
@@ -418,6 +418,8 @@ internal TraversalInfo NextLevel
     /// </summary>
     internal sealed class ComplexViewObjectBrowser
     {
+        private static readonly HashSet<string> s_defaultLeafTypes = new() { "System.DateTime" };
```

Comment:
`DateTimeOffset` is good candidate in the list.

Should the HashSet be OrdinalIgnorecase?

Reply:
I'm uncertain about casing - This seemed like the safest choice, but we could also ensure that we never have types that only differ by case in the set of types.